### ISSUE TITLE
Unable to install modules

### ISSUE THREAD

**Issue by stephanjohnson** [uilibrary/Octavia-admin-Pro-Issues#1]
Hi,

I've just downloaded a copy of this theme from the vuetify store and when I run `yarn install` I get the following output:

```
error An unexpected error occurred: "could not find a copy of eslint to link in /home/stephan/dev/themes/octavia/node_modules/eslint-config-vuetify/node_modules".
```

**Comment by naime-hossain** [comment]
Can you please try Npm Install instead? let me know after that. thank you

**Comment by naime-hossain** [comment]
I am updating all the packages,will update the product soon :) 
please try npm install for now or 
yarn cache clean
then 
yarn install
Thank you

**Comment by stephanjohnson** [comment]
Hi @naime-hossain ,

Some feedback:

`yarn cache clean` + `yarn install` did not work.

The NPM route did work. Thank you.

As a recommendation, until you've resolved the yarn issues, maybe remove the instructions for yarn from the readme.md file. 

PS: I do prefer yarn over NPM, so when it is working, please let me know.

**Comment by naime-hossain** [comment]
Thank you, I am working on yarn support :)

**Comment by naime-hossain** [comment]
For Now Solution :
`npm install `
or 
```
npm install --legacy-peer-deps

npm run serve

npm run build
```

**Comment by naime-hossain** [comment]
Yarn issue fixed, will update the theme in vuetify store soon, you can replace the package.json with this and try also remove the package.lock file.

`
{
    "name": "Octavia-Admin-pro",
    "version": "1.1.0",
    "private": true,
    "scripts": {
        "serve": "vue-cli-service serve",
        "build": "vue-cli-service build",
        "lint": "vue-cli-service lint"
    },
    "dependencies": {
        "core-js": "^3.6.5",
        "vue": "^2.6.11",
        "vue-router": "^3.2.0",
        "vuetify": "^2.4.0",
        "apexcharts": "^3.26.0",
        "axios": "^0.21.1",
        "firebase": "^8.3.1",
        "highlight.js": "^10.7.1",
        "nprogress": "^0.2.0",
        "vue-apexcharts": "^1.6.0",
        "vue-axios": "^3.2.4",
        "vue-chartjs": "^3.5.1",
        "vue-clickaway": "^2.2.2",
        "vue-form-wizard": "^0.8.4",
        "vue-i18n": "^8.24.2",
        "vue-instantsearch": "^3.6.0",
        "vue-meta": "^2.4.0",
        "vue-navigation-bar": "^4.1.0",
        "vue-perfect-scrollbar": "^0.2.1",
        "vuedraggable": "^2.24.3",
        "vuex": "^3.6.2",
        "vueperslides": "^2.13.0",
        "vue-map-chart": "0.0.2"
    },
    "devDependencies": {
        "axios-mock-adapter": "^1.19.0",
        "@mdi/font": "^4.8.95",
        "@vue/cli-plugin-babel": "~4.5.0",
        "@vue/cli-plugin-eslint": "~4.5.0",
        "@vue/cli-plugin-router": "~4.5.0",
        "@vue/cli-service": "~4.5.0",
        "@vue/eslint-config-standard": "^5.1.2",
        "@vuetify/vue-cli-plugin-preset-base": "~0.3.2",
        "babel-eslint": "^10.1.0",
        "eslint": "^6.7.2",
        "eslint-config-vuetify": "^0.4.1",
        "eslint-plugin-import": "^2.20.2",
        "eslint-plugin-node": "^11.1.0",
        "eslint-plugin-promise": "^4.2.1",
        "eslint-plugin-standard": "^4.0.0",
        "eslint-plugin-vue": "^6.2.2",
        "eslint-plugin-vuetify": "^1.0.0-beta.6",
        "lodash": "^4.17.14",
        "sass": "^1.32.0",
        "sass-loader": "^10.0.0",
        "vue-cli-plugin-vuetify": "latest",
        "vue-template-compiler": "^2.6.11",
        "vuetify-loader": "^1.7.0",
        "webfontloader": "^1.6.28"
    }
}

`

Thank you

**Comment by naime-hossain** [comment]
Hello, @stephanjohnson hope you are doing well, Yarn issue fixed, download the Octavia v1.6.0 from the store. Thank you
